docs/cloud-native-security/cspm-get-started-gcp.asciidoc

```diff
@@ -0,0 +1,82 @@
+[[cspm-get-started-gcp]]
+= Get started with CSPM for GCP
+
+[discrete]
+[[cspm-overview-gcp]]
+== Overview
+
+This page explains how to get started monitoring the security posture of your cloud assets using the Cloud Security Posture Management (CSPM) feature.
+
+.Requirements
+[sidebar]
+--
+* The CSPM integration is available to all {ecloud} users. On-premise deployments require an https://www.elastic.co/pricing[Enterprise subscription].
+* To view posture data, you need `read` privileges for the following {es} indices:
+** `logs-cloud_security_posture.findings_latest-*`
+** `logs-cloud_security_posture.scores-*`
+** `Logs-cloud_security_posture.findings`
+* The user who gives the CSPM integration GCP permissions must be a GCP project `admin`.
+--
+
+[discrete]
+[[cspm-setup-gcp]]
+== Set up CSPM for GCP
+
+To set up CSPM for GCP, first add the CSPM integration, then enable cloud account access.
+
+
+[discrete]
+[[cspm-add-and-name-integration-gcp]]
+=== Add your CSPM integration
+. From the Elastic Security *Get started* page, click *Add integrations*.
+. Search for `CSPM`, then click on the result.
+. Click *Add Cloud Security Posture Management (CSPM)*.
+. Give your integration a name that matches the purpose or team of the GCP account you want to monitor, for example, `dev-aws-account`.
+
+[discrete]
+[[cspm-set-up-cloud-access-section-gcp]]
+=== Set up cloud account access
+The CSPM integration requires admin access to your GCP project to
```

Comment:
> requires admin access

This is fine for the beta phase, however for GA, I have got a [task](https://github.com/elastic/security-team/issues/7503) to understand what are the specific privileges needed to run the cloud shell deployment.

Reply:
@benironside, we should make this read something like "Initially setting up CSPM for GCP will require you to have admin privileges in the project"

@uri-weisman, I want to clarify here that CSPM doesn't always require admin privileges. In fact, after setup, we use read-only access (correct me if this is wrong). This is also what we do with AWS.

Reply:
That is 100% correct @tinnytintin10